R/dd_loglik.R

```diff
@@ -212,40 +461,44 @@ dd_loglik1 = function(pars1,pars2,brts,missnumspec,methode = 'odeint::runge_kutt
             for(k in 2:(S + 2 - soc))
             {
               k1 = k + (soc - 2)
-              y = dd_integrate(probs,brts[(k-1):k],rhs_func_name,c(pars1,k1,ddep),rtol = reltol,atol = abstol,method = methode)
-              probs = y[2,2:(lx+1)]
+              #y = deSolve::ode(probs,brts[(k-1):k],rhs_func,c(pars1,k1,ddep),rtol = reltol,atol = abstol,method = "analytical")
+              #probs2 = y[2,2:(lx+1)]
+              probs = dd_loglik_M(pars1,lx,k1,ddep,tt = abs(brts[k] - brts[k-1]),probs)
```

Comment:
I'm unsure which version should be kept here

Reply:
No, this is just git mismatching the versions because of the large size of the file and the redundancies in the script. In reality, `new-dd-models` retains the same version as `develop`